server.js

```diff
@@ -2,6 +2,7 @@ var express = require('express'),
     app = express(),
     server = require('http').createServer(app),
     io = require('socket.io').listen(server),
+    sanitizehtml = require('./www/scripts/sanitizehtml.js')
```

Comment:
I'm still able to inject some `HTML tags` which could lead to `deface` :smile:

![Screenshot from 2020-09-04 17-52-30](https://user-images.githubusercontent.com/33063403/92260218-da88d000-eed7-11ea-9d8c-7b3d1a31edce.png)

Also, the `emoji` functionality doesn't work well :cry: 

Cheers,
Mik